packages/ui/package.json

```diff
@@ -50,7 +50,6 @@
     "./top-banner": "./components/top-banner/index.ts",
     "./filter-select": "./components/filter-select/index.ts",
     "./unpublished-entity/UnpublishedEntity": "./components/unpublished-entity/index.ts",
-    "./components/icon": "./components/icon/index.ts",
```

Comment:
we never import from `@calcom/ui/components/icon` currently + we already have `"./icon": "./components/icon/index.ts",` above (hence we should import from `@calcom/ui/icon`)